lib/chat/bloc/ViewModelProvider.dart

```diff
@@ -0,0 +1,9 @@
+import 'dart:async';
+
+abstract class ViewModelProvider<T> {
```

Comment:
this exists in the project already, how do we deal with that I wonder? Is the chat product in its own scope?

Reply:
Having Chat as something isolated (as SDK) maybe we should consider having this in two places. Otherwise we should add a dependency o start thinking in some new workaround for that. 
What is you opinion guys? @albertmiro @DavidFD005 @LeeMatthewHiggins

Reply:
That is not generating name conflicts, we would leave that like it is for now.